utils/modular_model_converter.py

```diff
@@ -841,12 +852,19 @@ def leave_SimpleStatementLine(self, original_node, updated_node):
                     self.all_imports.append(updated_node)
                 return updated_node
             elif m.matches(original_node, m.SimpleStatementLine(body=[m.Assign()])):
-                if original_node.body[0].targets[0].target.value in ASSIGNMENTS_TO_KEEP.keys():
+                value = original_node.body[0].targets[0].target.value
+                if value in ASSIGNMENTS_TO_KEEP.keys():
                     file_ = ASSIGNMENTS_TO_KEEP[original_node.body[0].targets[0].target.value]
                     self.files[file_][original_node.body[0].targets[0].target.value] = {
                         "node": original_node,
                         "insert_idx": self.global_scope_index,
                     }
+                elif any(re.search(pattern, value) for pattern in MODEL_DOCSTRING_PATTERNS):
+                    file_ = "modeling"
+                    self.files[file_][original_node.body[0].targets[0].target.value] = {
+                        "node": original_node,
+                        "insert_idx": self.global_scope_index,
+                    }
```

Comment:
the docstrings were not being ported because they were not from to "super-class" and the converter doesn't port assignments in any other place except for this one